.travis.yml

```
@@ -1,11 +1,15 @@
 language: python
+addons:
+  postgresql: "9.6"
+before_script:
+  - psql -c 'create database database;' -U postgres
+  - python3 manage.py makemigrations
+  - python3 manage.py migrate
 python:
   - "3.6"
-  - "3.6-dev"  # 3.6 development branch
-  - "3.7-dev"  # 3.7 development branch
 # command to install dependencies
 install:
   - pip install -r requirements.txt
 # command to run tests
 script:
-  - python manage.py test
+  - pytest
```

Новая строка в конце файла.

```suggestion
  - pytest

```